src/Okta.Sdk.IntegrationTests/InlineHooksClientShould.cs

```diff
@@ -0,0 +1,340 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Xunit;
+
+namespace Okta.Sdk.IntegrationTests
+{
+    public class InlineHooksClientShould
+    {
+        private const string SdkPrefix = "dotnet_sdk";
+        private const string InlineHookVersion = "1.0.0";
+        private const string InlineHookType = "com.okta.oauth2.tokens.transform";
+
+        private static readonly InlineHookChannel TestInlineHookChannel = new InlineHookChannel
```

Comment:
Nit: Based on our conversation [here](https://github.com/okta/okta-sdk-dotnet/pull/379#discussion_r427436591), let's create/define entities/values inside of each test.

Reply:
Updated.